submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss

```diff
@@ -0,0 +1,161 @@
+.header {
+padding:20px;
+background-color: #fff;
+}
+
+.header-nav__list {
+display: flex;
+justify-content: flex-end;
+align-items: center;
+gap: 25px;
+}
+
+.header-nav__item  {
+display: flex;
+justify-content: center;
+align-items: center;
+flex-basis: 30px;
+}
+
+.header-nav__link {
+display: flex;
+justify-content: center;
+align-items: center;
+text-decoration: none;
+color: rgb(17, 15, 15);
+transition: 250ms color cubic-bezier(0.075, 0.82, 0.165, 1);
+
+    &:hover,
+    &:focus { 
+    color: rgb(170, 63, 63);
+    }
+
+        &.link-img:hover {
+        box-shadow: 2px 2px 8px 2px rgba(18, 20, 22, 0.14);
+        }
+}
+
+.visually-hidden {
+position: absolute;
+appearance: none;
```

Comment:
Please, fix spacing

Reply:
i did`t know i need to fix all spacing when css file do it automatically when i write any property and use enter i see 1 tab from property to value and its really good seeing this better appearance: none; than this appearance:none;

Reply:
So if you don't. Please fix your code formatter in your vsCode. Code style is very important. 
You can add Prettier extention in code redactor.